source/brailleDisplayDrivers/hims.py

```diff
@@ -404,21 +407,24 @@ def _sendPacket(
 			# Data block 2 start
 			b"\xf2",
 			# Data block 1 length
-			d2Len.to_bytes(2, "little", signed=False),
+			d2Len.to_bytes(length=2, byteorder="little", signed=False),
 			# Data block 2
 			data2,
 			# Data block 2 end
 			b"\xf3",
 			# Reserved bytes
-			b"\x00"*4,
+			b"\x00" * 4,
 			# Reserved space for checksum
+			# Note that the checksum has the -3rd position in the final packet bytearray,
+			# whereas it has the -2nd position in the packet list
 			b"\x00",
 			# Packet end
 			b"\xfd"*2,
 		]
 		packetB = bytearray(b"".join(packet))
+		checksumIndexInPacketB: int = -3

```

Comment:
```suggestion
		#  checksum is the 3rd index from the end because 'packet end' takes up
		# two bytes and 'packetB' is a bytearray
		checksumIndexInPacketB: int = -3
```